src/components/main/index.js

```diff
@@ -1,35 +1,42 @@
-import React from "react";
+import React from 'react'
 
-import { Wrapper } from "./styles";
-import Nav from "components/nav";
-import { useLocation } from "react-router-dom";
+import { Wrapper } from './styles'
+import Nav from 'components/nav'
+import { useLocation } from 'react-router-dom'
 
 function getBackgroundImages(path) {
   const backgroundChangeList = {
-    "/create-wallet/status/updated": {
-      desktop: require("assets/images/desktop-purple-background.svg"),
-      mobile: require("assets/images/mobile-purple-background.svg"),
-      backgroundColor: "#503dc5"
+    '/create-wallet/status/updated/:consentNotGiven': {
+      desktop: require('assets/images/desktop-purple-background.svg'),
+      mobile: require('assets/images/mobile-purple-background.svg'),
+      backgroundColor: '#503dc5',
     },
-    "/create-wallet/created": {
-      desktop: require("assets/images/desktop-coviid-background.svg"),
-      mobile: require("assets/images/mobile-coviid-background.svg"),
-      backgroundColor: "#f1f1f8"
-    }
-  };
-
-  return backgroundChangeList[path] || null;
+
+    '/create-wallet/status/updated/:consentNotGiven': {
```

Comment:
no need to duplicate the same object. the param is dynamic and can be substituted with any value. remove one of these objects because you only need only one